### PR TITLE
Adding job state and job history status filter.

### PR DESCRIPTION
**Pull request by pinwang81**
Adding job state and job history status filter.

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@pinwang81__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (pinwang). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

**Comment by stankovski** [comment]
@yugangw-msft please merge when you are ready

**Comment by stankovski** [comment]
@pinwang81 please pull from upstream and update the PR

File: src/ResourceManagement/Scheduler/SchedulerManagement/Microsoft.Azure.Management.Scheduler.nuspec [hunk 1]
```diff
@@ -30,8 +30,8 @@
   </metadata>
   <files>
     <file src="ResourceManagement\Scheduler\SchedulerManagement\**\*.cs" target="src" />
-    <file src="..\binaries\net45\Microsoft.Azure.Management.Scheduler.dll" target="lib\dotnet" />
```
Comment (stankovski) [inline src/ResourceManagement/Scheduler/SchedulerManagement/Microsoft.Azure.Management.Scheduler.nuspec]: Please set the target to `target="lib\dotnet"` and src to `..\binaries\portable\xxx`

**Comment by pinwang81** [comment]
Any update on this?

**Pull request merged**